packages/xo-server-netbox/src/index.js

```diff
@@ -306,7 +306,7 @@ class Netbox {
 
     log.info('Synchronizing VMs')
 
-    const createNetboxVm = async (vm, { cluster, platforms }) => {
+    const createNetboxVm = async (vm, { cluster, platforms, netboxTags }) => {
```

Comment:
Can you find a better name than `netboxTags`?

Also, it make the code a bit harder to understand with `cluster` vs `netboxTags`.

Reply:
I'm trying to differentiate Netbox objects from XO objects even when there's no variable name conflict (`tag` vs `netboxTag`). But when the object type was already different, I didn't namespace them (`pool` vs `cluster`). But maybe I should? And maybe I should also namespace XO objects?

Reply:
Yes, I like it. It might be a bit verbose but at least it's clear and consistent.